### ISSUE TITLE
When switch themes it gives an error about my theme file.

### ISSUE THREAD

**Issue by Enise115** [hyprwm/Hyprland#8674]
When switch themes it is continuously giving me an error drop_shadow not defined or doesn't exist.
Everything else is working fine.

When fixing this with nano config.... 
It's completely gone and no error is showing. 
But when I switch themes it comes back. 

I changed the majority of my files that could possibly give the problem when switching themes. 
That unfortunately didn't fix the problem. 

Does anyone have the same problem?
If so. 
Love to hear it.

**Comment by Nicknamely** [comment]
It should tell you which line and file.
And check [breaking changes](https://hyprland.org/news/update45/) while you are at it. 

> decoration:shadow_ stuff has moved

**Comment by Enise115** [comment]
"It should tell you which line and file.
And check breaking changes while you are at it."

I haven't check the breaking changes 
Thanks, i will look into it.

It's only when I'm switching themes.
Ill change config/hypr/themes/theme.conf 
Drop_shadow
To 
#drop_shadow
Problem solved!

But, when switch themes it showing again which is what i expect.
That's why i couldn't the error in the other files.

**Comment by vaxerski** [comment]
that's a problem with your theme not hyprland

**Comment by Enise115** [comment]
> that's a problem with your theme not hyprland

My error is stating that the problem is with the global file ~/.config/hypr/hyprland.conf

**Comment by vaxerski** [comment]
that's **your** config, fix it yourself.

**Comment by Enise115** [comment]
> that's **your** config, fix it yourself.

Okay. I don't understand your frustration towards me. 

I just started my Linux journey 3 months ago. 
Mistakes can happen especially when people are just starting. 

I just simply stated if anyone knows why this problem is coming up would love to know.

**Comment by Enise115** [comment]
> > that's **your** config, fix it yourself.
> 
> Okay. I don't understand your frustration towards me. 
> 
> I just started my Linux journey 3 months ago. 
> Mistakes can happen especially when people are just starting. 

The problem within the theme file was easy to solve 
But unfortunately I wasn't able to solve the global file ~/.config/hypr/hyprland.conf 
it’s likely due to default or global configs applied by EndeavourOS that might override.
 Again I'm not sure. 
> 
> I just simply stated if anyone knows why this problem is coming up would love to know. 
> 
>

**Comment by Nicknamely** [comment]
Were you able to fix it ?

**Comment by Enise115** [comment]
Yeas
But I'm using my own wm now!

**Comment by Nicknamely** [comment]
You made your own wm ?! Mind sharing ?  I wanna learn.

**Comment by Enise115** [comment]
> You made your own wm ?! Mind sharing ?  I wanna learn.

İ didn't fully made it myself 
Let me start with that ;).

İ personally didnt really use hypr at all. 
I'm not the demographic audience that its actually meant for. And personally I haven't had the most  positive interactions with other people using it. 

Still very new to the whole applying and creating your own environment. Let alone to the whole Linux environment. 
My only encounter with any of this was OpenOffice. 

My friends started a small project that has the idea of a whole wm. But it is still very unstable...
We also are considering creating our own display server, and even communication protocol. 

That's where my interest took over into the whole communication protocols. 

Basically i wanted to do it myself as well. 

İt really helps me to understand more about the  creation and structural changes/behaviour.

**Comment by Enise115** [comment]
İll recommend looking for libraries that you can implement in your source code.

Also ask yourself what you want to do with it. 
Some project are huge have over more than 100.000 lines of code. And some literally just 50 lines. 

Using other window manager as reference can help alot. 
İ3 and DWM have inspired majority of my friends.
